app/js/features/escrow/saga.js

```diff
@@ -120,7 +121,55 @@ export function *onLoadEscrows() {
   yield takeEvery(LOAD_ESCROWS, doLoadEscrows);
 }
 
+
+export function *checkUserRating({address}) {
+  try {
+    address = address || web3.eth.defaultAccount;
+    const state = yield select();
+    const offers = Object.values(state.metadata.offers).filter(offer => offer.owner.toLowerCase() === address.toLowerCase());
+    if (!offers.length) {
+      return yield put({type: USER_RATING_SUCCEEDED, userRating: -1, address});
```

Comment:
in case of no offer should it be
```
downCount: 0, upCount: 0, voteCount: 0
```

Reply:
Oops, my bad, that's the remnant of an old rating calculation I had